images.go

```diff
@@ -486,6 +523,17 @@ func (r *imageStore) BigDataNames(id string) ([]string, error) {
 	return image.BigDataNames, nil
 }
 
+func imageSliceWithoutValue(slice []*Image, value *Image) []*Image {
```

Comment:
Pretty horrible name.
removeImage()?
pruneImage()?

Reply:
It'd probably be `pruneImageFromSlice()` or something like that, since we're not changing anything else.  This name is intentionally similar to `stringSliceWithoutValue` from `store.go`.

Reply:
Ok, not crazy about the name, but if it consistently sucks...